code/__HELPERS/time.dm

```diff
@@ -4,25 +4,14 @@
 #define HOURS   *36000
 
 //Returns the world time in english
-proc/worldtime2text(time = world.time)
+/proc/worldtime2text(time = world.time)
 	return "[round(time / 36000)+12]:[(time / 600 % 60) < 10 ? add_zero(time / 600 % 60, 1) : time / 600 % 60]"
 
-proc/time_stamp()
+/proc/time_stamp()
 	return time2text(world.timeofday, "hh:mm:ss")
-
-/* Preserving this so future generations can see how fucking retarded some people are
-proc/time_stamp()
-	var/hh = text2num(time2text(world.timeofday, "hh")) // Set the hour
-	var/mm = text2num(time2text(world.timeofday, "mm")) // Set the minute
-	var/ss = text2num(time2text(world.timeofday, "ss")) // Set the second
-	var/ph
-	var/pm
-	var/ps
```

Comment:
Why did you remove this? The comment specifically stated it was there as an example of terribad code.